### PR TITLE
optional escape htmlspecialcharacter parameter for link_to_route

### PR DESCRIPTION
**Pull request by paanblogger**

### CODE REVIEW

File: src/HtmlBuilder.php [hunk 1]
```diff
@@ -220,9 +220,9 @@ public function linkSecureAsset($url, $title = null, $attributes = [])
      *
      * @return \Illuminate\Support\HtmlString
      */
-    public function linkRoute($name, $title = null, $parameters = [], $attributes = [])
+    public function linkRoute($name, $title = null, $parameters = [], $attributes = [] , $escape = true)
     {
-        return $this->link($this->url->route($name, $parameters), $title, $attributes);
+        return $this->link($this->url->route($name, $parameters), $title, $attributes , null , $escape);
```
Comment (mlantz) [inline src/HtmlBuilder.php]: I see an extra null here

Comment (paanblogger) [inline src/HtmlBuilder.php]: I edit this based on the : 
`public function link($url, $title = null, $attributes = [], $secure = null, $escape = true)`

**Comment by mlantz** [comment]
Please review the failing checks

File: src/HtmlBuilder.php (hunk 1 above)
Comment (mlantz) [inline src/HtmlBuilder.php]: Let's add in the option for secure too - I don't see any reason to not have that option.

**Comment by tshafer** [comment]
I don't like to pass booleans into functions.

**Pull request closed without merging**